src/organizercore.cpp

```diff
@@ -364,6 +371,11 @@ void OrganizerCore::downloadRequestedNXM(const QString &url)
   }
 }
 
+void OrganizerCore::userInterfaceInitialized(QMainWindow* mainWindow) 
```

Comment:
I don't think `mainWindow` is necessary, it's already passed in `setUserInterface()`.

Reply:
`setUserInterface` gets a `IUserInterface*` which is an internal interface (not exposed through `uibase`).

Reply:
My point was that `OrganizerCore` already has a pointer to `IUserInterface` so it doesn't need to be given the main window again. There's `IUserInterface::qtWidget()` you can use, and you can change it to return a `QMainWindow` if you need to.

We also typically use events for this sort of stuff, instead of having `MainWindow` call into `OrganizerCore` directly. But I don't really care.

Reply:
Should be "fixed" in 8c410af